hazelcast/include/hazelcast/client/internal/config/xml_config_locator.h

```diff
@@ -0,0 +1,89 @@
+/*
+* Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+ */
+#pragma once
+
+#include <string>
+#include <vector>
+#include "hazelcast/util/export.h"
+#include <boost/property_tree/ptree.hpp>
+#include "hazelcast/client/client_config.h"
+#include "hazelcast/client/client_properties.h"
+
+#if defined(WIN32) || defined(_WIN32) || defined(WIN64) || defined(_WIN64)
+#pragma warning(push)
+#pragma warning(disable : 4251) // for dll export
+#endif
+
+namespace hazelcast {
+namespace client {
+namespace config {
```

Comment:
```suggestion
namespace internal {
namespace config {
```

Namespace should also be internal as well as the file location.

Reply:
done.